### PR TITLE
Feature/aql subquery execution block impl execute implementation k shortest path executor

### PR DESCRIPTION
**Pull request by markuspf**
Internal Team Gordo PR for review

### CODE REVIEW

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8293/

**Comment by mchacki** [comment]
new Jenkins: (after merge, old seemed to be unhappy)
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8378/

File: arangod/Graph/KShortestPathsFinder.cpp [hunk 1]
```diff
@@ -39,6 +39,8 @@
 #include <velocypack/StringRef.h>
 #include <velocypack/velocypack-aliases.h>
 
+#include "Logger/LogMacros.h"
```
Comment (mchacki) [inline arangod/Graph/KShortestPathsFinder.cpp]:
```suggestion

```

or keep it, if you implement a debug logging pattern

File: arangod/Aql/KShortestPathsExecutor.cpp [hunk 2]
```diff
@@ -40,6 +40,8 @@
 
 #include <utility>
 
+#include "Logger/LogMacros.h"
```
Comment (mchacki) [inline arangod/Aql/KShortestPathsExecutor.cpp]: You still log here?
If not please remove

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8387/

**Comment by markuspf** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8399/

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8420/

**Comment by mchacki** [comment]
Jenkins Again:
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8460/

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8691/

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8701/

**Comment by markuspf** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8712/

**Pull request merged**